src/HeapedVectors.jl

```diff
@@ -0,0 +1,165 @@
+__precompile__()
+
+module HeapedVectors
+
+import Base: getindex, length, push!, isempty,
+        pop!, resize!, popfirst!
+
+using IntervalArithmetic
+export Strategy,heap,HeapedArray,heapVectors,SortedArray,sorVectors 
+
+abstract type Strategy end
+struct heapVectors <:Strategy end
```

Comment:
These should not be necessary. You only need `SortedVector` and `Heap`, i.e. the names of the Julia types themselves.

Reply:
All  these keywords which are mentioned in export statements are all needed in `optimise.jl`. thats why they are needed here.

Reply:
You have `heap,HeapedArray,heapVectors`.
As far as I can see, it should only be necessary to have one of these names, which is the name of the type itself.